examples/image-test-workflow/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,172 @@
+          - name: REPO
+            default: https://github.com/codefreshdemo/cf-example-unit-tests-with-composition
+          - name: REVISION
+            default: master
+          - name: IMAGE_NAME
+          # Create git secret token and apply it to your cluster under the runtime namespace
+          #   kubectl create secret generic github-token \
+          #   --from-literal=token=$GIT_TOKEN --dry-run=client \
+          #   --save-config -o yaml | kubectl apply -f - -n $NAMESPACE
+          - name: GIT_TOKEN_SECRET
+            default: github-token # the name of the git secret token you applied to your cluster
+          - name: DOCKERFILE
+            default: Dockerfile
+          - name: CONTEXT
+            default: .
+          - name: TAG
+            default: latest
+          
+      dag:
+        tasks:
+          # build a docker image through kaniko
+          # https://codefresh.io/argohub/workflow-template/kaniko
+          - name: build-docker-image
+            templateRef:
+              name: argo-hub.kaniko.0.0.1
+              template: build
+            arguments:
+              parameters:
+                - name: REPO
+                  value: '{{ inputs.parameters.REPO }}'
+                - name: REVISION
+                  value: '{{ inputs.parameters.REVISION }}'
+                - name: IMAGE_NAME
+                  value: '{{ inputs.parameters.IMAGE_NAME }}'
+                - name: GIT_TOKEN_SECRET
+                  value: '{{ inputs.parameters.GIT_TOKEN_SECRET }}'
+                - name: DOCKERFILE
+                  value: '{{ inputs.parameters.DOCKERFILE }}'
+                - name: CONTEXT
+                  value: '{{ inputs.parameters.CONTEXT }}'
+                - name: TAG
+                  value: '{{ inputs.parameters.TAG }}'
+
+          # create mysql service container
+          - name: create-service-container
+            dependencies: [build-docker-image]
+            template: mysql-service
+            arguments:
+              parameters:
+                - name: MYSQL_SECRET
+                  value: '{{ inputs.parameters.MYSQL_SECRET }}'
+                - name: MYSQL_DATABASE
+                  value: '{{ inputs.parameters.MYSQL_DATABASE }}'
+                - name: MYSQL_HOST
+                  value: '{{ inputs.parameters.MYSQL_HOST }}'
+
+          # intialize mysql service container and test
+          - name: initialize-service-container
+            dependencies: [create-service-container]
+            template: mysql-client
+            arguments:
+              parameters:
+                - name: IMAGE
+                  value: '{{ inputs.parameters.IMAGE_NAME }}:{{ inputs.parameters.TAG }}'
+
+    - name: mysql-service
+      metadata:
+        annotations:
+          argo-hub-template/description: 'Build service container for MYSQL'
+          argo-hub-template/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/image-test-workflow/assets/icon.svg"
```

Comment:
need to be change to this "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/image-test-workflow/assets/icon.svg"